packages/extension-api/src/extension-api.d.ts

```diff
@@ -1500,7 +1500,7 @@ declare module '@podman-desktop/api' {
     /**
      * The resource the dialog shows when opened.
      */
-    defaultUri?: Uri;
+    defaultUri?: Uri | string;
```

Comment:
Can't we keep it as Uri only and assume that it is a file uri ?

Reply:
What do you mean exactly? Bc i would have to call the openDialog from the client. So i would have something like `window.openDialog(mypath)`. I could create a fake Uri myself with 
```
scheme: file,
path: mypath,
fspath: mypath, 
....
```
but it's a waste imo. In the openDialog it's going to take the fsPath in any case

Reply:
I meant the API is already designed to open a file or folder so why do you have to update the API ?

Reply:
bc there was no way to create a URI item from the frontend and bc the defaultUri can be basically be a string i was extending it. I'm going to create another PR to expose the createUri function